rosdep/base.yaml

```
@@ -3780,6 +3780,9 @@ libmodbus5:
   gentoo: [dev-libs/libmodbus]
   nixos: [libmodbus]
   ubuntu: [libmodbus5]
+libwebsockets-dev:
+  debian: [libwebsockets-dev]
```

It would be really great if you would take the time to check around for support on other platforms before opening the PR so that we don't have to. It makes reviewing the change a lot easier. This one is available all over the place...

Alpine: https://pkgs.alpinelinux.org/package/edge/main/x86_64/libwebsockets-dev
Arch: https://archlinux.org/packages/extra/x86_64/libwebsockets
Fedora: https://src.fedoraproject.org/rpms/libwebsockets#bodhi_updates
Gentoo: https://packages.gentoo.org/packages/net-libs/libwebsockets
NixOS: https://search.nixos.org/packages?channel=21.05&show=libwebsockets&from=0&size=50&sort=relevance&type=packages&query=libwebsockets
OpenSUSE: https://software.opensuse.org/package/libwebsockets
RHEL (via EPEL): https://src.fedoraproject.org/rpms/libwebsockets#bodhi_updates

```suggestion
  alpine: [libwebsockets-dev]
  arch: [libwebsockets]
  debian: [libwebsockets-dev]
  fedora: [libwebsockets-devel]
  gentoo: [net-libs/libwebsockets]
  nixos: [libwebsockets]
  opensuse: [libwebsockets-devel]
  rhel: [libwebsockets-devel]
```